monai/transforms/intensity/dictionary.py

```diff
@@ -164,7 +167,7 @@ def randomize(self, im_shape: Sequence[int]) -> None:
         for m in self.mean:
             self._noise.append(self.R.normal(m, self.R.uniform(0, self.std), size=im_shape))
 
-    def __call__(self, data: Mapping[Hashable, np.ndarray]) -> Dict[Hashable, np.ndarray]:
+    def __call__(self, data: Mapping[Hashable, Any]) -> Dict[Hashable, Any]:
```

Comment:
`Mapping[Hashable, Any]` doesn't really tell us anything since the first type has to be hashable. This and the `Dict` version should be stored as pre-defined types maybe.

Reply:
What do you think those pre-defined types should look like? e.g., `NdarrayTensorDict = Dict[Hashable, NdarrayTensor]`. The problem here is that it implies that all values of the dictionary are of type `NdarrayTensor`, which isn't true. Or is it OK since the only elements we'll deal with in this function are of type `NdarrayTensor`?

Reply:
for readability, I updated it to `def __call__(self, data: Mapping[Hashable, NdarrayTensor]) -> Dict[Hashable, NdarrayTensor]`